tests/lib/rules/no-unused-disable.js

```diff
@@ -77,7 +77,7 @@ describe("no-unused-disable", () => {
         const selfPath = path.resolve(__dirname, "../../../")
         const pluginPath = path.resolve(
             __dirname,
-            "../../../node_modules/eslint-plugin-eslint-comments"
+            "../../../node_modules/@eslint-community/eslint-plugin-eslint-comments"
```

Comment:
I think we probably need to create a `""../../../node_modules/@eslint-community/"` directory after this.